pom.xml

```diff
@@ -982,6 +982,7 @@
                             <useFile>false</useFile>
                             <argLine>
                                 -Xms128m -Xmx1G -XX:MaxPermSize=1024M
+                                -XX:+HeapDumpOnOutOfMemoryError
```

Comment:
I prefer aligning additional JVM arguments with master branch and using the `${extraVmArgs}` across the file.

Reply:
That sounds like a good idea, didn't think of it. With that it should be possible to configure the `-XX:+HeapDumpOnOutOfMemoryError` via the build configuration. Good one!